pkg/tftarget/tftarget_test.go

```diff
@@ -0,0 +1,270 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tftarget
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/config-validator/pkg/targettesting"
+	"github.com/open-policy-agent/frameworks/constraint/pkg/client"
+)
+
+// match creates a match struct as would exist in a FCV constraint
+func match(opts ...func(map[string]interface{})) map[string]interface{} {
+	var matchBlock = map[string]interface{}{}
+	matchBlock["resource_address"] = map[string]interface{}{}
+
+	for _, opt := range opts {
+		opt(matchBlock)
+	}
+
+	var destructured interface{} = matchBlock
+	return destructured.(map[string]interface{})
+}
+
+func stringToInterface(s []string) []interface{} {
```

Comment:
Since this is directly copied can we put it into the targettesting package?

Reply:
done